crates/compiler/parse/src/ast.rs

```diff
@@ -539,6 +541,15 @@ pub enum Expr<'a> {
     OptionalFieldInRecordBuilder(&'a Loc<&'a str>, &'a Loc<Expr<'a>>),
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum EmptyBlockParent {
```

Comment:
Shouldn't this information be derivable from where the empty block exists in the syntax tree?